tests/queries/models.py

```diff
@@ -264,6 +264,14 @@ class Related(models.Model):
     custom = models.ForeignKey(CustomPk, models.CASCADE)
 
 
+class RelatedNull(models.Model):
+    custom = models.ForeignKey(CustomPk, models.CASCADE, null=True)
```

Comment:
Is there a problem adding null=True` on the `Related` model? We try to avoid adding more models since it increases the run time of the test suite.